Modules/ModParser.lua

```diff
@@ -3416,6 +3417,9 @@ local count = 0
 --foo:close()
 return function(line, isComb)
 	if not cache[line] then
+		if line:match("you are hexproof") then
```

Comment:
What's the purpose of this block?

Reply:
I accidentally left it in when debugging